hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1572,6 +1738,16 @@ int SaraNcpClient::initMuxer() {
     return SYSTEM_ERROR_NONE;
 }
 
+int SaraNcpClient::startNcpFwUpdate(bool update) {
+    if (ncpId() != PLATFORM_NCP_SARA_R510) {
+        return SYSTEM_ERROR_NONE;
+    }
+
+    firmwareUpdateR510_ = update;
```

Comment:
@technobly If my understanding is correct, this branch does not support the firmware updates. How to best test/understand that this won't do any firmware updates?